src/org/parosproxy/paros/extension/manualrequest/http/impl/HttpPanelSender.java

```diff
@@ -311,4 +322,17 @@ public URI getInvalidRedirection() {
             return invalidRedirection;
         }
     }
+    
+    private void setUseTrackingSessionState(boolean shouldUseTrackingSessionState) {
+        if (!shouldUseTrackingSessionState) {

```

Comment:
This if can be removed from this method (should now be done in `setButtonTrackingSessionStateEnabled`).

Reply:
In `setButtonTrackingSessionStateEnabled` I want to set it no matter what...

I wanted to separate the enable and selected setting as the button needs to be set disabled when initially displayed, but not when the user presses the button.

Reply:
I'm not following, the button needs to be deselected when the button is disabled (to effectively disable the option).